### ISSUE TITLE
Stub column scope

### ISSUE THREAD

**Issue by ouelletv** [johannheyne/acf-table-field-pro-support#19]
Hi there! Thanks for the great plugin.

I created a site which has been submitted to and accessibility audit. They indicated me that the "Stub column" had a scope="col" and that it should be  scope="row" instead and i think it makes sens. Is there something im missing here or it is indeed an issue?

(Screenshots are in french, sorry for that)
Thanks!

<img width="795" height="342" alt="Image" src="https://github.com/user-attachments/assets/d2aeece6-467c-47e8-be0a-8cea8cf0b083" />
<img width="1274" height="486" alt="Image" src="https://github.com/user-attachments/assets/701dc9a6-3bf6-43e6-b794-ea5a877ee679" />
<img width="1850" height="430" alt="Image" src="https://github.com/user-attachments/assets/42337a6f-7d99-4800-a154-404c1be80659" />

**Comment by johannheyne** [comment]
Hi, 

Yes, I agree—it does seem like the scope attribute should be set to "row" in this case. However, this attribute isn’t generated by the Table Field plugin itself; it was likely added either via a custom PHP filter in the Theme or by a feature of an accessibility plugin.

The Table Field plugin PHP filter through which the scope attribute could be added is the following:
https://www.acf-table-field.com/docs/filters/main/get-table-attr-cell/latest/
This filter allows you to modify the HTML attributes of table cells generated by the Table Field plugin, making it possible to inject custom attributes like scope="row".

Hope this clears things up for you!
Johann

**Comment by ouelletv** [comment]
You are totally right. We have a11y snippets in our codebase and one of them was causing the issue. 
I will look towards this. 

Thank you very much and sorry for the inconvenience!
